lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -174,16 +174,16 @@ def connected?
 
       private
 
-      def declare_exchange!(channel, exchange, exchange_type, durable)
+      def declare_exchange!(channel, exchange, exchange_type, durable, arguments)
```

Comment:
This changes an API called by the other plugins, and it's unclear whether or not that will cause breakages. For example, if the upgrade to a newer (but not one that includes your input PR) input plugin, and somehow this plugin is upgraded, that plugin will fail to work because it will be calling `declare_exchange!` with the incorrect parameters.